src/pages/FrontendMentor/ElvaYBPractice/styles/SinglePriceGridComponent.styled.ts

```diff
@@ -0,0 +1,136 @@
+import styled from 'styled-components'
+
+export const MainPage = styled.div`
+  min-height: 100vh;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(225, 217, 221, 0.8);
+`
+
+export const MainContainer = styled.div`
+  width: 70%;
+  border-radius: 15px;
+  overflow: hidden;
+`
+export const BottomContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  @media (max-width: 768px) {
```

Comment:
You can follow the width breakpoint that frontend mentor has given. The page seems slightly off.